templates/rest-boot-api/js-client/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "${projectName}-client",
+    "version": "${version}",
+    "main": "./v1/apiClientClient.js"
+}
```

Comment:
Here we should add the following dependency 
popsicle": "^0.3.6
Maybe this package should be generated by the Javascript generator.